docs/node-configuration.md

```diff
@@ -47,14 +47,19 @@ DBConfiguration:
   Type: leveldb
   LevelDBOptions:
     DataDirectoryPath: /chains/privnet
+    ReadOnly: false
   BoltDBOptions:
     FilePath: ./chains/privnet.bolt
+    ReadOnly: false
 ```
 where:
 - `Type` is the database type (string value). Supported types: `levelDB` and
   `boltDB`.
-- `LevelDBOptions` are settings for LevelDB.
-- `BoltDBOptions` configures BoltDB.
+- `LevelDBOptions` are settings for LevelDB. Includes the DB files path and ReadOnly mode toggle.
+  If ReadOnly mode is on, then an error will be return on attempt to connect to unexisting or empty
```

Comment:
`be returned`, and it's not the main thing, the main thing is that it won't be changed (at the same time I don't know how a real node would react to that now).

Reply:
> the main thing is that it won't be changed

I thought that it's clear that RO mode doesn't allow changes.

> I don't know how a real node would react to that now

Nothing critical, it just fails to persist changes:

```
2022-10-07T15:35:51.407+0300	WARN	failed to persist blockchain	{"error": "leveldb: read-only mode"}
2022-10-07T15:35:51.520+0300	INFO	started protocol	{"addr": "159.223.15.210:21333", "userAgent": "/NEO-GO:0.99.3/", "startHeight": 851609, "id": 1886857966}
2022-10-07T15:35:51.526+0300	INFO	started protocol	{"addr": "49.12.229.165:20333", "userAgent": "/Neo:3.4.0/", "startHeight": 797799, "id": 2123125405}
2022-10-07T15:35:51.538+0300	WARN	peer disconnected	{"addr": "104.215.144.118:20333", "error": "already connected", "peerCount": 13}
2022-10-07T15:35:51.544+0300	INFO	started protocol	{"addr": "52.230.99.53:40333", "userAgent": "/Neo:3.4.0/", "startHeight": 851609, "id": 1433067073}
2022-10-07T15:35:52.408+0300	WARN	failed to persist blockchain	{"error": "leveldb: read-only mode"}
2022-10-07T15:35:53.412+0300	WARN	failed to persist blockchain	{"error": "leveldb: read-only mode"}
q2022-10-07T15:35:54.413+0300	WARN	failed to persist blockchain	{"error": "leveldb: read-only mode"}
2022-10-07T15:35:55.414+0300	WARN	failed to persist blockchain	{"error": "leveldb: read-only mode"}
2022-10-07T15:35:56.415+0300	WARN	failed to persist blockchain	{"error": "leveldb: read-only mode"}
```

Reply:
I'll add a note about it.

Reply:
Fixed.